From 4767894600906cd55ce7e43e1a142d617b82fbd8 Mon Sep 17 00:00:00 2001
From: Sebastian Krzyszkowiak <dos@dosowisko.net>
Date: Thu, 12 Sep 2019 22:50:24 +0000
Subject: [PATCH] Change the minimum window size to 350x350

This allows to run the game on smaller screens, such as phones.
---
 src/game-window.vala | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/game-window.vala b/src/game-window.vala
index 8ff248e..c8aef57 100644
--- a/src/game-window.vala
+++ b/src/game-window.vala
@@ -123,8 +123,8 @@ private class GameWindow : ApplicationWindow
     * * window state
     \*/
 
-    private const int WINDOW_MINIMUM_SIZE_HEIGHT = 600;
-    private const int WINDOW_MINIMUM_SIZE_WIDTH = 600;
+    private const int WINDOW_MINIMUM_SIZE_HEIGHT = 350;
+    private const int WINDOW_MINIMUM_SIZE_WIDTH = 350;
 
     private int _window_width;
     private int _window_height;
-- 
2.24.1

